causallearn/search/FCMBased/PNL/PNL.py

```diff
@@ -181,28 +164,28 @@ def cause_or_effect(self, data_x, data_y):
         pval_forward: p value in the x->y direction
         pval_backward: p value in the y->x direction
         '''
+        torch.manual_seed(0)
```

Comment:
why we need to fix random seed in your algorithm?

Reply:
In our method, we take the neural network to learn/approximate the nonlinear relationship.